src/ui/public/pattern_checker/pattern_checker.html

```diff
@@ -1,2 +1,2 @@
 <span class="spinner"></span>
-Querying {{checker.pattern}}... {{checker.resultCount}} results
```

Comment:
Not a blocker for this PR; I think that this line is very important to what is happening on this page, and that the way it is styled/located right now causes it to be lost... @alt74 what do you think? I know where it is now, and know to watch it, but the first few times that I used the wizard, it wasn't obvious. Nor was the link between that line and the 'progress' bar at the top of the page.